src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -140,22 +96,9 @@
       </v-expand-transition>
     </section>
 
-    <!-- Agreement Type -->
-    <template v-if="isTypeBcCcc">
-      <section class="mt-10">
-        <header id="confirm-agreement">
-          <h2>2. Confirm Incorporation Agreement and Article Completion</h2>
-        </header>
-
-        <v-card flat class="mt-6">
-          <AgreementType :showErrorSummary="!getIncorporationAgreementStep.valid" />
-        </v-card>
-      </section>
-    </template>
-
     <!-- Sample Templates -->
-    <template v-else>
-      <section class="mt-10">
+    <template>
+      <section v-if="isTypeBcomp || isTypeBcCompany || isTypeBcCcc" class="mt-10">
```

Comment:
Question for this part. There is an updated ULC share structure sample to down, so why is the ULC type omitted from this section tag?

Reply:
The ULC type doesn't have more text after this point where as the rest of the corp types have more tooltips and text.

Reply:
This is the "Sample Templates" section to download the sample PDF file. Omitting the ULC entity type here will hide this section for the ULC entity to download the sample file.

Reply:
I don't know, I think you are right the download for the sample template. However I think that a CCC's like ULC's can only have custom incorporation agreements. I'll add this to the list of questions for Yui.